src/Config/ConfigLoader.php

```diff
@@ -32,12 +33,13 @@ function load(string $workingDir): Config
 
         $data = \array_replace_recursive(self::DEFAULTS, $data);
 
-        $data['image'] !== null
-            or U::fail('Image name is not specified');
+        $data['image'] !== null or $data['container'] !== null
```

Comment:
```suggestion
        $data['image'] !== null || $data['container'] !== null
```

`or` is only used for the error throwing part due to its low operator priority.